x-pack/plugins/serverless/public/types.ts

```diff
@@ -24,6 +26,7 @@ export interface ServerlessPluginStart {
   setNavigation(projectNavigation: ChromeProjectNavigation): void;
   setProjectHome(homeHref: string): void;
   setSideNavComponent: (navigation: SideNavComponent) => void;
+  getActiveNavigationNodes$: () => Observable<ChromeProjectNavigationNode[][]>;
```

Comment:
I followed the convention in place for the Chrome service (https://github.com/elastic/kibana/blob/main/packages/core/chrome/core-chrome-browser-internal/src/chrome_service.tsx#LL365C7-L425C71). A function returning an observable.

Reply:
I will create a stable reference of the `Observable` before returning it 👍

Reply:
Or to follow the pattern, keep the getter on service level, but pass an observable to the component like here: https://github.com/elastic/kibana/blob/1bb316d97fbf0fcd39f25a66c188ba2694de6d96/packages/core/chrome/core-chrome-browser-internal/src/chrome_service.tsx#L330

Reply:
I think the only "problem" is because I decided to use `.asObservable()`. I am going to pass down the subject and it should be ok. That's how it is done _most_ of the time... but sometimes not... you know, consistency! 😊 

```ts
navLinks$={navLinks.getNavLinks$()}
recentlyAccessed$={recentlyAccessed.get$()}
navControlsLeft$={navControls.getLeft$()}
navControlsCenter$={navControls.getCenter$()}
navControlsRight$={navControls.getRight$()}
navControlsExtension$={navControls.getExtension$()}
...
```

Reply:
Fixed in https://github.com/elastic/kibana/pull/159906/commits/74318d7d1f8828b60fb4d79dd0982a4dfcf021b6